hdr/sqlite_modern_cpp.h

```diff
@@ -91,6 +91,10 @@ namespace sqlite {
 			}
 			
 		}
+		void endr() {
+		  execute();
+		  reset();
+		}
 		
```

Comment:
We already have the operator, so I don't think we need a member function. If we keep this function, we may prefer a more descriptive name.